src/App.css

```diff
@@ -0,0 +1,18 @@
+.article-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.article-card {
+  border: 3px solid black;
+  width: 15rem;
+  list-style-type: none;
+  margin-bottom: 20px;
+}
+
+.card-image {
+  width: 50%;
+  flex: 1;
+}
```

Comment:
Don't forget to consider spacing in your app to make is user friendly! 👀 Nice work on making your app responsive!